draft-ietf-cbor-7049bis.md

```diff
@@ -904,7 +904,7 @@ Tag number 1 contains a numerical value counting the number of seconds
 from 1970-01-01T00:00Z in UTC time to the represented point in civil
 time.
 
-The tag content data item MUST be an unsigned or negative integer (major types 0
```

Comment:
I occasionally used “tag content data item” for emphasis that the tag content is a data item.  Too much redundancy?

Reply:
Yes, definitely.